apps/graphql/src/apps/itinerary/Itinerary.js

```diff
@@ -19,20 +19,12 @@ type SearchLocation = {|
   +ids: string[],
 |};
 
-export type ItinerariesCommonSearchParameters = {|
-  +travelFrom: string[],
-  +dateFrom: Date,
-  +dateTo?: Date,
-  +travelTo?: string[],
-  +passengers?: Passengers,
-|};
-
 export type ItinerariesOneWaySearchParameters = {|
   +order?: Order,
   +sort?: Sort,
   +itinerary: {|
     +origin: SearchLocation,
-    +destination?: SearchLocation,
+    +destination: SearchLocation,
```

Comment:
The `+destination: SearchLocation,` is now required?

Reply:
It was to circumvent the Flow issue with the union, but is now fixed. Placed it back!